code/modules/surgery/organs/eyes.dm

```diff
@@ -5,17 +5,19 @@
 	organ_tag = "eyes"
 	parent_organ = "head"
 	slot = "eyes"
-	var/eye_colour = "#000000" // Should never be null
+	var/eye_color = "#000000" // Should never be null
 	var/list/colourmatrix = null
 	var/list/colourblind_matrix = MATRIX_GREYSCALE //Special colourblindness parameters. By default, it's black-and-white.
 	var/list/replace_colours = GREYSCALE_COLOR_REPLACE
 	var/dependent_disabilities = list() //Gets set by eye-dependent disabilities such as colourblindness so the eyes can transfer the disability during transplantation.
-	var/weld_proof = null //If set, the eyes will not take damage during welding. eg. IPC optical sensors do not take damage when they weld things while all other eyes will.
+	var/weld_proof = FALSE //If set, the eyes will not take damage during welding. eg. IPC optical sensors do not take damage when they weld things while all other eyes will.
 
 	var/vision_flags = 0
 	var/see_in_dark = 2
+	var/tint = 0
+	var/flash_protect = FLASH_PROTECTION_NONE
 	var/see_invisible = SEE_INVISIBLE_LIVING
-	var/lighting_alpha = LIGHTING_PLANE_ALPHA_VISIBLE
+	var/lighting_alpha
```

Comment:
This is `null` by default to be consistent with glasses and helmet `light_alpha`; this has no changes to how mobs see, because base mobs have this defined on themselves.